spimdisasm/mips/symbols/MipsSymbolBase.py

```diff
@@ -67,7 +67,7 @@ def canUseConstantsOnData(self) -> bool:
         return self.contextSym.allowedToReferenceConstants
 
 
-    def generateAsmLineComment(self, localOffset: int, wordValue: int|None=None, *, isDouble: bool=False, emitRomOffset: bool=True) -> str:
+    def generateAsmLineComment(self, localOffset: int, wordValue: Union[int|list]|None=None, *, isDouble: bool=False, isString: bool=False, emitRomOffset: bool=True) -> str:
```

Comment:
Both `Union` and `|`?

Reply:
The other way around, avoid using Union.
```suggestion
    def generateAsmLineComment(self, localOffset: int, wordValue: int|list[int]|None=None, *, isDouble: bool=False, isString: bool=False, emitRomOffset: bool=True) -> str:
```